runtime/lua/vim/filetype.lua

```diff
@@ -1528,32 +1528,32 @@ local pattern = {
   ['.*/etc/xdg/menus/.*%.menu'] = 'xml',
   ['.*Xmodmap'] = 'xmodmap',
   ['.*/etc/zprofile'] = 'zsh',
-  ['%.bash[_-]aliases'] = function()
-    vim.fn['dist#ft#SetFileTypeSH']('bash')
+  ['%.bash[_-]aliases'] = function(path, bufnr)
+    require('vim.filetype.detect').sh(path, bufnr, 'bash')
   end,
-  ['%.bash[_-]logout'] = function()
-    vim.fn['dist#ft#SetFileTypeSH']('bash')
+  ['%.bash[_-]logout'] = function(path, bufnr)
+    require('vim.filetype.detect').sh(path, bufnr, 'bash')
   end,
-  ['%.bash[_-]profile'] = function()
-    vim.fn['dist#ft#SetFileTypeSH']('bash')
+  ['%.bash[_-]profile'] = function(path, bufnr)
+    require('vim.filetype.detect').sh(path, bufnr, 'bash')
   end,
-  ['%.cshrc.*'] = function()
-    vim.fn['dist#ft#CSH']()
+  ['%.cshrc.*'] = function(path, bufnr)
+    require('vim.filetype.detect').csh(path, bufnr)
   end,
   ['%.gtkrc.*'] = starsetf('gtkrc'),
-  ['%.kshrc.*'] = function()
-    vim.fn['dist#ft#SetFileTypeSH']('ksh')
+  ['%.kshrc.*'] = function(path, bufnr)
+    require('vim.filetype.detect').sh(path, bufnr, 'ksh')
   end,
-  ['%.login.*'] = function()
-    vim.fn['dist#ft#CSH']()
+  ['%.login.*'] = function(path, bufnr)
+    require('vim.filetype.detect').csh(path, bufnr)
   end,
   ['%.neomuttrc.*'] = starsetf('neomuttrc'),
-  ['%.profile.*'] = function()
-    vim.fn['dist#ft#SetFileTypeSH'](vim.fn.getline(1))
+  ['%.profile.*'] = function(path, bufnr)
+    require('vim.filetype.detect').sh(path, bufnr, vim.fn.getline(1))
```

Comment:
`vim.fn.getline(1)` should be `getline(bufnr, 1)` (same for all other instances in this file)